governance/internationals.md

```diff
@@ -0,0 +1,55 @@
+# International Bible Quizzing
+These rules apply to the Internationals quiz meet conducted once yearly, administered under the oversight of the CQLT.
+
+## Eligibility Requirements
+- All quizzers for International competition must be 12 to 18 years of age by the meet registration date or have been enrolled as a full-time high school student during the current quiz year.
+- In order to quiz at Internationals for a specific district, the quizzer must reside in that district or in a bordering community.  Their home church must be from the district in which they quiz.  Any exceptions to this must be brought to the C&MA Quizzing Leadership Team (CQLT).
+
+## Team Structure
+- Each district will be allowed to send as many teams as the district feels will be competitive and that they are financially able to send.  These will be the district International Teams.
+- A team will be composed of a maximum of five members: four starting quizzers and one substitute.  These five shall constitute the team for International Finals.  Any exceptions to this must be brought to the C&MA Quizzing Leadership Team (CQLT).
+- Each team must have a coach and may also have an assistant coach. 
+
+## Event Registration
+- All arrangements, promotion, and administration will be by the Life Impact Office.
```

Comment:
Zach should chime in on this one, but is this true in terms of actually what happens? LIM handles arrangements, but the CQLT handles promotion, right?

Reply:
yeah, I just brought this stuff over. Not sure if this doc is the best place for it.

Reply:
I handle registration by creating the forms and then sending it out to everyone. The LIFE Office takes care of communicating that information as necessary to the facility. Jordan also uses that information to create the booklets containing the rosters. Though I typically send that to her as well. 

On an overall note: Life Impact Ministries no longer exists and was changed to The LIFE Office. They went from a separate but symbiotic organization to a department/ministry in the National Office. I would suggest swapping all references to LIM to The LIFE Office.

Reply:
OH, so they're BACK inside the national office? LOL

Reply:
but Zach, do you think this line needs to exist in the Internationals.md doc? Or is it better defined in a different spot?

Reply:
I think this needs to be defined somewhere, and it seems reasonable to put it in the Internationals doc since it relates to that meet.